velox/functions/prestosql/aggregates/ApproxPercentileAggregate.cpp

```diff
@@ -324,13 +324,17 @@ class ApproxPercentileAggregate : public exec::Aggregate {
         BaseVector::wrapInConstant(numGroups, 0, std::move(array));
     rowResult->childAt(kPercentilesIsArray) =
         std::make_shared<ConstantVector<bool>>(
-            pool, numGroups, false, BOOLEAN(), bool(percentiles_->isArray));
```

Comment:
This was not a cast. This was a constructor which could be redundant.
Same below.

Reply:
@aditi-pandit Thanks for clarifying. If the intent is to get an r-value, let's use `std::move()` or `static_cast<bool&&>`.

Reply:
Yes, good suggestion. Using static_cast<bool&&> is better.